manifests/05-helm.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: helm.openshift.io/v1beta1
+kind: HelmChartRepository
+metadata:
+  name: redhat-helm-repo
+spec:
+  name: Red Hat Helm Charts
+  connectionConfig:
+    url: https://redhat-developer.github.io/redhat-helm-charts
```

Comment:
So, 
1. All authenticated users get to see helm chart repos
2. Ootb, they see the default one. https://redhat-developer.github.io/redhat-helm-charts was previously hard-coded in the console config - instead of giving it any special treatment, it is being made just another CR. 👍

Reply:
> Ootb, they see the default one. https://redhat-developer.github.io/redhat-helm-charts was previously hard-coded in the console config

you mean in the `console-config` CM in `openshift-console` namespace ?

Reply:
HelmChartRepository is cluster scoped.